packages/server/src/api/todo/social/postNotificationsSeen.ts

```diff
@@ -0,0 +1,32 @@
+import { Server } from '../../../lexicon'
+import { AuthRequiredError, InvalidRequestError } from '@adxp/xrpc-server'
+import * as util from '../../../util'
+import { User } from '../../../db/user'
+
+export default function (server: Server) {
+  server.todo.social.postNotificationsSeen(async (_params, input, req, res) => {
+    const { seenAt } = input.body
```

Comment:
this allows any time to be set as `seenAt` i thought maybe doing the `Max` of `seenAt` & the currently registered time in the DB, but then thought there might be a usecase for "rewinding" notifications 🤔